cypress/integration/played_cards_spec.js

```diff
@@ -0,0 +1,37 @@
+describe('The Played Cards Component', function() {
+    beforeEach(() => {        
+        cy.visit('/')
+        const username = "unicorn"
+        cy.get('input').type(username)
+        cy.get('button').click()
+        /* Connect a second user using our fake client */
+        const url = Cypress.config().baseUrl;
+        cy.request(`http://localhost:8081/connect-and-play?url=${encodeURIComponent(url)}`)
+    })
+
+    afterEach(() => {
+        cy.request({
+            url: '/api/reset-server',
+            method: 'POST'
+        });
+        cy.visit('/')
+    });
+
+    const cardsNumber = 3;
+    const numberOfPlayers = 2;
+
+    /* GAME STARTS AND PLAYERS ARE GIVEN CARDS */
+    describe("on second player's turn, after first player has played", () => {
+        it('player is prompted to pick a card and all the cards are displayed on the end of his turn', () => {
```

Comment:
Surprised you went for "his".
```suggestion
        it('player is prompted to pick a card and all the cards are displayed on the end of their turn', () => {
```